BHoM_Adapter/BHoM_Adapter.csproj

```diff
@@ -26,6 +26,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
+    <PackageReference Include="System.Collections.Immutable" Version="8.0.0" />
```

Comment:
Is this needed? Would not add new nugets if not required.

Reply:
@IsakNaslundBh, not needed anymore.
Thanks for flagging this. 
Now removed from the code.